samples/net/dns_resolve/README.rst

```diff
@@ -13,6 +13,14 @@ If a CNAME is received, the DNS resolver will create another DNS query.
 The number of additional queries is controlled by the
 DNS_RESOLVER_ADDITIONAL_QUERIES Kconfig variable.
 
+The multicast DNS (mDNS) client resolver support can be enabled by setting
+CONFIG_MDNS_RESOLVER Kconfig variable.
```

Comment:
you can generate a link reference to a Kconfig options by using ``:option:`CONFIG_MDNS_RESOLVER` ``